queries.sql

```diff
@@ -8,3 +8,25 @@ select name, escape_attemps from animals where weight_kg > 10.5;
 select * from animals where neutered= true;
 select * from animals where name != 'Gabumon';
 select * from animals where weight_kg between 10.3 and 17.4;
+
+start transaction;
+update animals set species = 'unspecified';
+rollback;
+start transaction;
+update animals set species='digimon' where name like '%mon';
+update animals set species = 'pokemon' where species = 'NULL';
+commit,
+start transaction;
+delete from animals;
+rollback;
+start transaction;
```

Comment:
- [ ] Great job on implementing these queries as requested however, kindly make sure you use the correct commands. instead of start transaction you may consider saying `BEGIN` or `BEGIN TRANSACTION`

```suggestion
begin;
update animals set species = 'unspecified';
rollback;
begin;
update animals set species='digimon' where name like '%mon';
update animals set species = 'pokemon' where species = 'NULL';
commit,
begin;
delete from animals;
rollback;
start transaction;
```

Reply:
@christianbiring1  please I'm using mySQL

Reply:
In MySQL, you can use either START TRANSACTION or BEGIN TRANSACTION to start a transaction. Both statements have the same effect of beginning a transaction block.
Thanks @christianbiring1

Reply:
well resolved! ✔️